eclair-core/src/main/scala/fr/acinq/eclair/db/pg/PgChannelsDb.scala

```diff
@@ -246,6 +253,35 @@ class PgChannelsDb(implicit ds: DataSource, lock: PgLock) extends ChannelsDb wit
     }
   }
 
+  override def listClosedChannels(remoteNodeId_opt: Option[PublicKey], paginated_opt: Option[Paginated]): Seq[PersistentChannelData] = withMetrics("channels/list-closed-channels", DbBackends.Postgres) {
+    val sql = "SELECT data FROM local.channels WHERE is_closed=TRUE ORDER BY closed_timestamp"
```

Comment:
Don't we want to order by `closed_timestamp DESC`? We want the most recently closed channels first, don't we?